source/_posts/2023-01-30-php-core-roundup-9.md

```diff
@@ -0,0 +1,428 @@
+
+### RFC Updates
+
+Following are the RFCs and major pull-requests discussed, voted, and implemented since our last update.
+
+
+
+
+*	**RFC In Voting: [Readonly amendments](https://wiki.php.net/rfc/readonly_amendments) 💜**
+
+	RFC by Nicolas Grekas and Máté Kocsis, attempts to address some of the shortcomings of PHP 8.1 readonly properties and 8.2 readonly classes.
+
+	This RFC proposes allowing `readonly` classes to be extended by non-readonly classes (currently not allowed, and causes a fatal error), and to allow reinitializing readonly properties during cloning (within the `__clone()` magic method).
+
+*	**RFC Accepted: [More Appropriate Date/Time Exceptions](https://wiki.php.net/rfc/datetime-exceptions) 💜**
+
+	RFC by Derick Rethans, proposed to introduce Date/Time extension-specific exceptions and errors. This detailed RFC suggests more specificity in the exceptions with exception classes such as `DateInvalidTimeZoneException`, and `DateMalformedPeriodStringException` as well as promoting some of the current PHP warnings to Error exceptions.
+
+    The vote was accepted, and is pending implementation.
+
+
+*	**RFC Implemented: [Randomizer Additions](https://wiki.php.net/rfc/randomizer_additions)**
+
+	RFC by Joshua Rüsweg and Tim Düsterhus, proposed to add new “building block” methods to `\Random\Randomizer` (added in PHP 8.2) that implement commonly useful operations that are either verbose or very hard to implement in userland.
+
+	The RFC proposed to add two new methods: `getBytesFromString`, to generate a random string containing specific characters, and `getFloat`/`nextFloat`, to generate a random floating point value.
+
+*	**RFC Under Discussion: [Pass Scope to Magic Accessors](https://wiki.php.net/rfc/pass_scope_to_magic_accessors) 💜**
+
+	RFC by Nicolas Grekas and Ilija Tovilo, proposing to pass the calling scope to magic accessors to make it trivial to get it. While it is currently possible to retrieve the calling scope by retrieving the backtrace, the RFC proposes to pass the calling scope as the last argument to the magic methods, making it easier compared to fiddling with the debug backtrace.
+
+*	**RFC Under Discussion: [Saner array_(sum|product)()](https://wiki.php.net/rfc/saner-array-sum-product) 💜**
+
+	RFC by George Peter Banyard suggests improvements to the existing array_sum and array_product functions by emitting a warning when the arrays being summed/multiplied contain invalid types. This can introduce backwards-incompatible effects (apart from the warning) on code that relies on the current behavior of the functions.
+
+*	**RFC Under Discussion: [Add file_descriptor() function](https://wiki.php.net/rfc/file-descriptor-function) 💜**
+
+	RFC by George Peter Banyard proposes a new `file_descriptor()` function that returns the integer file descriptor of a given stream, if the underlying file system supports it.
```

Comment:
Out of all the ones I've proposed, you skipped the Increment/Decrement one which is probably the bigger/more controversial one. .-.

Also SameSite cookie as a parameter, but that one is small.

Reply:
Ah good catch, thank you. I will add them.